### PR TITLE
Fix for #8

### PR DESCRIPTION
**Pull request by milovanderlinden**

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -80,7 +80,9 @@
 		});
 		return uuid;
 	}
-	document.getElementById('boxurl').value = 'https://jsonbox.io/box_' + create_UUID();
+	var base = window.location.protocol + '//' + window.location.hostname;
+	base += (window.location.port === 80 || window.location.port) === 433 ? '' : ':' + window.location.port;
```
Comment (hg-pyun) [inline www/index.html]: Need to 443

Comment (milovanderlinden) [reply www/index.html]: Fixed

File: www/index.html [hunk 2]
```diff
@@ -80,7 +80,9 @@
 		});
 		return uuid;
 	}
-	document.getElementById('boxurl').value = 'https://jsonbox.io/box_' + create_UUID();
+	var base = window.location.protocol + '//' + window.location.hostname;
+	base += (window.location.port === 80 || window.location.port) === 443 ? '' : ':' + window.location.port;
```
Comment (vasanthv) [inline www/index.html]: There is an issue with this implementation. I get an extra `:` in the URL. 
Sample `https://jsonbox.io:/box_129326b6cacf71fb5f14`.

**Comment by vasanthv** [comment]
@milovanderlinden thanks for the PR. Can you fix the issue I mentioned in the review? I can merge it after that.

**Pull request merged**